packages/infra/lib/shared/rds.ts

```diff
@@ -2,19 +2,28 @@ import * as cloudwatch from "aws-cdk-lib/aws-cloudwatch";
 import { SnsAction } from "aws-cdk-lib/aws-cloudwatch-actions";
 import * as rds from "aws-cdk-lib/aws-rds";
 import { Construct } from "constructs";
-import { RDSAlarmThresholds } from "../../config/aws/rds";
 import { mbToBytes } from "../shared/util";
-
+import { EnvConfig } from "../../config/env-config";
 const DEFAULT_MIN_LOCAL_STORAGE_MB_ALARM = 10_000;
 
+const getMaxPostgresConnections = (maxAcu: number): number => {
```

Comment:
Let's not use arrow functions, please - [ref](https://www.notion.so/metriport/Development-Best-Practices-bb3574d3c83d46dc9914a9e5c5542ca7?pvs=4#b669ec3473f1414b9d234cbdac20041b)